KinSDK/KinSDK/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.9.1</string>
```

Comment:
Why downgrade?

Reply:
1.0 is the default when starting a new project. It was never set, I have no set it.